<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cbo_select</class>
 <widget class="QWidget" name="cbo_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>312</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_user">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yahei</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:pressed
{
    color:white;
/*    background-color:rgb(14 , 135 , 228);*/
	background-color:rgb(41, 144, 163);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="icon">
      <iconset resource="imageIcon.qrc">
       <normaloff>:/imgIcon/appbar.scrabble.u.png</normaloff>:/imgIcon/appbar.scrabble.u.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>76</width>
       <height>76</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yahei</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
     </property>
     <property name="text">
      <string>姓名    </string>
     </property>
     <property name="icon">
      <iconset resource="imageIcon.qrc">
       <normaloff>:/imgIcon/appbar.scrabble.n.png</normaloff>:/imgIcon/appbar.scrabble.n.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>76</width>
       <height>76</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_sex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yahei</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
     </property>
     <property name="text">
      <string>性别    </string>
     </property>
     <property name="icon">
      <iconset resource="imageIcon.qrc">
       <normaloff>:/imgIcon/appbar.scrabble.s.png</normaloff>:/imgIcon/appbar.scrabble.s.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>76</width>
       <height>76</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_height">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yahei</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BlankCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:pressed
{
    color:white;
    background-color:rgb(14 , 135 , 228);
    border: 1px solid rgb(12 , 138 , 235);
    padding-left:3px;
    padding-top:3px;
}
</string>
     </property>
     <property name="text">
      <string>身高    </string>
     </property>
     <property name="icon">
      <iconset resource="imageIcon.qrc">
       <normaloff>:/imgIcon/appbar.scrabble.h.png</normaloff>:/imgIcon/appbar.scrabble.h.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>76</width>
       <height>76</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imageIcon.qrc"/>
 </resources>
 <connections/>
</ui>
